mmdet3d/datasets/pipelines/loading.py

```diff
@@ -666,3 +673,71 @@ def __repr__(self):
         repr_str += f'{indent_str}with_bbox_depth={self.with_bbox_depth}, '
         repr_str += f'{indent_str}poly2mask={self.poly2mask})'
         return repr_str
+
+
+@PIPELINES.register_module()
+class MultiViewPipeline(object):
+    """Load and transform multi-view images.
```

Comment:
Load is not included in this pipeline?

Reply:
Are you suggesting to split my `MultiViewPipeline` to 2 separate classes`MultiViewPipeline` and `LoadMultiViewImageFromFilesV2`? Not sure that adding 2 new transforms is better then 1. But if you think it is better, I can do it.

Reply:
Also we can move `to_float32`, `color_type`, and `file_client_args` from `LoadImageFromFile.__init__` to `MultiViewPipeline.__init__` and remove `LoadImageFromFile` from `MultiViewPipeline.transforms`. However, it is not beautiful too.

I prefer to add a comment and assert about the necessity of `LoadImageFromFile` in `MultiViewPipeline.transforms`.

Reply:
@ZwwWayne Please have a look at this function and see whether there is a conflict with your multi-modality models on nuScenes.